dprof.js

```diff
@@ -147,7 +147,17 @@ asyncHook.enable();
 // Print result
 //
 
-process.on('exit', function () {
+if (process.argv.indexOf('--raw-sigint') === -1 ||
```

Comment:
No, the logic is correct -- I think it is better UX to have it as the default, as I suspect that is what most folks will expect.

That's also why I called it `--raw-sigint`, as in, don't touch the default.

It could be something like `--no-sigint-handler` or `--no-dprof-sigint`.

The ENV variable exists for programs that may error on new flags.